database/data_seeding/nosql_seeder.js

```diff
@@ -0,0 +1,48 @@
+const { user, password } = require('../../database_configs/nosql_database.config.js');
+const nano = require('nano')(`http://${user}:${password}@localhost:5984`);
+const { generateData } = require('./data_generator.js');
+
+const destroyDb = async (dbName) => {
+  await nano.db.destroy(dbName);
+}
+
+const createDb = async (dbName) => {
+  await nano.db.create(dbName)
```

Comment:
between lines 9 and 10, you could put "await nano.db.destroy(dbName)" instead of having it be a separate function up above. However, what that would do, is every time you create a database, it will first delete the database. I am not sure if you want to do that but if you wanted to have 2 lines less code, you can add the delete database line before create database. Just something to consider.